src/Doctrine/Orm/Filter/ExpressionFilter.php

```diff
@@ -32,12 +33,13 @@ public function __construct(
         LoggerInterface $logger = null,
         array $properties = null,
         array $filters = null,
-        NameConverterInterface $nameConverter = null
+        NameConverterInterface $nameConverter = null,
+        bool $innerJoinsLeft = false
```

Comment:
Some filters depend on the inner joins. We can move instead to the FilterExpressionProvider to allow the users configure the desired filter instead of all filters.